.github/workflows/build.yaml

```diff
@@ -28,8 +28,8 @@ jobs:
           - tedge-rauc
           - tedge-mender
         machine:
-          - raspberrypi
-          - raspberrypi2
+          # RaspberryPi 64 bit: 1 2
```

Comment:
armv7 is actually the raspberry pi boards in 32 bit mode, here is the text taken from the [openembedded layer index site](https://layers.openembedded.org/layerindex/branch/master/machines/?q=raspberry&search=1). "RaspberryPi boards in 32 bit mode"

Though after digging a bit deeper, I'm now not so sure that `armv7` will cover Raspberry Pi 1 and 2...so we'll definitely need to test the compatibility here.